.cirrus.yml

```diff
@@ -210,6 +210,7 @@ task:
     docker_arguments:
       CI_IMAGE_NAME_TAG: ubuntu:jammy
       FILE_ENV: "./ci/test/00_setup_env_win64.sh"
+  << : *CREDITS_TEMPLATE
```

Comment:
this is unrelated to switching to the docker file, right? would prefer this change in it's own PR because I have a few questions about cost (before / after, etc), but not a super big deal